### PR TITLE
feat(atomic): improve positioning of modal and wait for app to load before showing it

### PR DESCRIPTION
**Pull request by liviarett**
This solves the issues with loading and scrolling seen in the video 

https://user-images.githubusercontent.com/30511433/182371026-0b03c559-b620-42c7-86ce-897cacf9e0d6.mov

It also works if your interface was to be positioned absolutely

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+name: Bug Report
```
Comment (btaillon) [inline .github/ISSUE_TEMPLATE/bug_report.yml:1]: 👍

**Comment by olamothe** [comment]
🤔 I think the diff is polluted with old pull requests

**Comment by liviarett** [comment]
fixed @olamothe @ThibodeauJF

**Review by ThibodeauJF — APPROVED** [review]

File: packages/atomic/src/components/insight/atomic-insight-refine-modal/atomic-insight-refine-modal.tsx [hunk 2]
```diff
@@ -54,6 +63,14 @@ export class AtomicInsightRefineModal
   private interfaceDimensions?: InsightInterfaceDimensions;
   private facetManager!: InsightFacetManager;
   private resizeObserver?: ResizeObserver;
+  private scrollCallback?: () => void;
+
+  @Listen('wheel')
+  @Listen('touchmove')
+  @Listen('keydown')
+  handleScroll() {
+    this.updateDimensions();
```
Comment (olamothe) [inline packages/atomic/src/components/insight/atomic-insight-refine-modal/atomic-insight-refine-modal.tsx:72]: Should there be some kind of debounce, either here or in the `updateDimensions` function ?

Does it need to run on every scroll event ? Just adding 50 or 100ms debounce might go a long way for perf.

**Review by olamothe — APPROVED** [review]

File: packages/atomic/src/components/insight/atomic-insight-refine-modal/atomic-insight-refine-modal.tsx (hunk 2 above)
Comment (liviarett) [reply packages/atomic/src/components/insight/atomic-insight-refine-modal/atomic-insight-refine-modal.tsx:72]: good point, I'll add it!

**Review by btaillon — APPROVED** [review]

**Pull request merged**